nilearn/plotting/img_plotting.py

```diff
@@ -37,6 +39,43 @@
 # Core, usage-agnostic functions
 
 
+def check_threshold(thr, data):
+    """
+    Checks that the given threshold has an accepted string value
+
+    Parameters
+    ----------
+    thr: str or a list of strings or number or list of numbers
```

Comment:
Maybe something like "real values, percentage express in a string or a list of these"